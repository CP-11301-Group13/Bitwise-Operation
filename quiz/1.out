65389 65422 65515 65417 65370 65388 65373 65464 65423 65380 65297 65337 65490 65385 65413 65388 65382 65494 65340 65360 65317 65404 65444 65508 65390 65520 65364 65303 65287 65362 65301 65398 65495 65401 65529 65480 65481 65402 65478 65426 65509 65430 65287 65437 65387 65313 65367 65334 65519 65381 65441 65351 65393 65447 65409 65428 65291 65382 65391 65441 65522 65338 65380 65506 65303 65322 65368 65527 65499 65362 65482 65410 65290 65334 65315 65522 65503 65375 65350 65414 65432 65461 65486 65403 65282 65330 65345 65327 65295 65467 65323 65339 65466 65280 65349 65435 65461 65376 65401 65290