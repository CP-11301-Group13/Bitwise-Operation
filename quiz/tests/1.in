54 1
37497 23292 61768 39545 5617 24086 11944 13067 45678 10833 18235 52799 55545 23025 13619 35477 55260 28773 33679 39520 39212 49253 26234 49770 50061 4276 49797 38741 46058 13933 23666 59162 45487 43823 13101 13554 3203 45125 25403 12665 54651 49128 45920 34761 36574 52955 29698 65072 42658 9036 62645 21768 5393 12949