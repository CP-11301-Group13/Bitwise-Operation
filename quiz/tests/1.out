35660 35660